doc/modules/changes/20180625_niday

```diff
@@ -0,0 +1,3 @@
+Fixed: The default latitude and longitude limits for the 'uniform radial' particle generator were set to (0, pi) even though they were measured in degrees.
```

Comment:
Please add how they are measured now, otherwise the reader will not know what is the fixed version.